apply/apply.go

```diff
@@ -68,7 +69,14 @@ func execPipeline(ctx context.Context, in *graph.Graph, pipelineF MkPipelineF, r
 		notify.Transform(func(id string, out *graph.Graph) error {
 			renderingPlant.Graph = out
 			pipeline := pipelineF(out, id)
-			val, pipelineError := pipeline.Exec(out.Get(id))
```

Comment:
What would that look like? I'm having a hard time understanding what you're saying here.